pytuya/__init__.py

```diff
@@ -293,7 +293,11 @@ def extract_payload(self,data):
         start=data.find(b'{"devId')
         if(start!=-1):
             result = data[start:] #in 2 steps to deal with the case where '}}' is present before {"devId'
-            end=result.find(b'}}')+2
+            end=result.find(b'}}')
+            if(end==-1):
```

Comment:
I am not surprise that you did not encountered that case. Without the persistent socket this case should not occur like that. You probably observe something related. I suspect that the encoded status payload you got is probably not a complete payload but just the dps with the first key. In my understanding, this is sometimes send by the device  (part of the protocol) and you got that message instead of the status.